package.json

```diff
@@ -19,29 +22,33 @@
   },
   "homepage": "https://github.com/seek-oss/eslint-config-seek#readme",
   "dependencies": {
-    "@babel/core": "^7.17.8",
-    "@babel/eslint-parser": "^7.17.0",
-    "@babel/preset-react": "^7.16.7",
-    "@typescript-eslint/eslint-plugin": "^5.16.0",
-    "@typescript-eslint/parser": "^5.16.0",
+    "@babel/core": "^7.19.6",
+    "@babel/eslint-parser": "^7.19.1",
+    "@babel/preset-react": "^7.18.6",
+    "@typescript-eslint/eslint-plugin": "^5.41.0",
+    "@typescript-eslint/parser": "^5.41.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-import-resolver-node": "^0.3.6",
+    "eslint-import-resolver-typescript": "3.5.2",
     "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-jest": "^27.0.1",
-    "eslint-plugin-react": "^7.29.4",
-    "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.1.3",
+    "eslint-plugin-react": "^7.31.10",
+    "eslint-plugin-react-hooks": "^4.6.0",
     "find-root": "^1.1.0"
   },
   "devDependencies": {
-    "@changesets/cli": "2.17.0",
-    "@changesets/get-github-info": "0.5.0",
-    "eslint": "8.11.0",
-    "prettier": "2.4.1",
-    "typescript": "^4.1.2"
+    "@changesets/cli": "2.25.0",
+    "@changesets/get-github-info": "0.5.1",
+    "eslint": "8.26.0",
+    "prettier": "2.7.1",
+    "typescript": "^4.8.4"
   },
   "peerDependencies": {
     "eslint": ">=6",
     "typescript": ">=3.3"
-  }
+  },
+  "engines": {
+    "node": "^16.18.0 || ^18.12.0"
```

Comment:
I believe SEEK have a lot of repos (indirectly) using this config on Node.js 14 😞, so if we enforce this it will have to be in a `major` version

Reply:
It's an unintended side-effect. I wanted to have something for [`node-version-file`](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file), so I'll change it to `volta`.